### PR TITLE
Make the quickstart actions accessible to screen reader and improve accessibility of some UI elements

### PR DESCRIPTION
**Pull request by daniv-msft**

### CODE REVIEW

**Comment by daniv-msft** [comment]
FYI, I used as screen reader NVDA, which was the one we used for our tests in Paris.
The default Windows narrator has the issues that:
- It has multiple bugs.
- It isn't compatible with Chrome.
- It represents a very small market share of screen readers. #ByDesign

**Review by zr-msft — APPROVED** [review]
Overall LGTM. Nice work @daniv-msft !

File: BikeSharingWeb/components/Field.js [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
 const Field = (props) => (
     <div>
-        <div className="fieldLabel">{props.label}</div>
-        <div className="fieldValue">{props.value}</div>
+        <div className="fieldLabel" tabIndex="0">{props.label}</div>
+        <div className="fieldValue" tabIndex="0">{props.value}</div>
```
Comment (DrEsteban) [inline BikeSharingWeb/components/Field.js:7]: >0 [](start = 46, length = 1)

Without having any clue what I'm talking about, shouldn't this be `1` / count up?  There are a few instances in other files as well. #Resolved

**Review by DrEsteban — APPROVED** [review]
:shipit:

File: BikeSharingWeb/components/Field.js (hunk 1 above)
Comment (daniv-msft) [reply BikeSharingWeb/components/Field.js:7]: Actually, it depends of what we want to do. :)
If we were giving tabIndex >= 1, then we would indicate the browser to follow our specified tab order.
Here, we indicate that the item is tabbable but by providing "0", we let the browser determine the order, so basically it follows the DOM which is what we want.

---
In reply to: [276479848](https://github.com/ContosoBikeRental/BikeSharingSampleApp/pull/35#discussion_r276479848) [](ancestors = 276479848)

**Pull request merged**